Donation-platform/src/main/java/com/google/SortByTrendingScore.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google;
+
+import com.google.model.Charity;
+import java.util.Comparator;
+
+class SortByTrendingScore implements Comparator<Charity> {
```

Comment:
why do we need a class to declare this function? If its being used in multiple classes then declare this class as Util

Reply:
it seems that for collections.sort(), which I use to sort the charities by trendingScore in FindTrendingCharities.java, I need a separate class that implements Comparator on the object I wish to sort by a custom metric (in this case trendingScore). Since it is only used it one line of code, I didn't think it would be necessary to make a Utils class.

Reply:
I've actually modified the Charity.java class so that it contains the compareTo() method needed for Collections.sort(), so I'll delete SortByTrendingScore.java.

Reply:
sg